### PR TITLE
Add blog post: Scala.js no longer experimental.

### PR DESCRIPTION
**Pull request by sjrd**
This should be posted on February 5, right after the official announcement of Scala.js 0.6.0. See PR https://github.com/scala-js/scala-js-website/pull/114
The first link in the very first paragraph will appear dead until that time.

### CODE REVIEW

**Comment by sjrd** [comment]
I would gladly appreciate comments of any kind, including to improve the text or the structure.

Review by @heathermiller 
/cc @adriaanm @gkossakowski @odersky

**Comment by heathermiller** [comment]
Great! I'll have a look and will add a commit to this PR with little grammatical fix-ups. Is that OK with you? Of course, I'll leave it to you to merge this when you guys are ready :)

**Comment by sjrd** [comment]
Sure, OK.

**Comment by sjrd** [comment]
Oh, cc @gzm0 as well. I guess you don't watch this repo ^^

File: news/_posts/2015-02-05-scala-js-no-longer-experimental.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+---
+
+Today, we [announced the release of Scala.js v0.6.0](http://www.scala-js.org/news/2015/02/05/announcing-scalajs-0.6.0/), the Scala to JavaScript compiler, and dropped the experimental flag associated to it.
+Yes, you read it right: Scala.js is no longer experimental!
+After exactly 2 years of development, we can finally call it production-ready.
+
+## More than a compiler, an entire ecosystem
+
+At its core, Scala.js is a compiler from Scala to JavaScript.
+It compiles .scala files to one optimized and minimized .js file per application.
```
Comment (gzm0) [inline news/_posts/2015-02-05-scala-js-no-longer-experimental.md]: to a single

File: news/_posts/2015-02-05-scala-js-no-longer-experimental.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+We've asked developers who use Scala.js daily why they think Scala.js is awesome.
+Here is a distilled version of what they have answered, in decreasing order of how many people mentioned it.
+You can read [the entire discussion on the mailing list](https://groups.google.com/forum/#!topic/scala-js/_1Sfb5Nj08w).
+
+* 5 Share code between client and server
+* 4 Strong typing, including for JavaScript libraries
+* 4 Tooling: IDE support, sbt integration, dependency, unit testing, stack traces and source maps, cross-compiling projects
+* 3 Integration with JavaScript
+* 4 Portability: bring Scala to environments powered by JavaScript, most importantly the browsers, but also others, such as the brand new [React Native](https://www.youtube.com/watch?v=KVZ-P-ZI6W4)
```
Comment (gzm0) [inline news/_posts/2015-02-05-scala-js-no-longer-experimental.md]: Ordering issue here?

Comment (sjrd) [inline news/_posts/2015-02-05-scala-js-no-longer-experimental.md]: Ah yes, indeed. I merged two similar 2's in one 4 and forgot to move it up ^^

**Comment by gzm0** [comment]
Sweet all that :)

**Comment by sjrd** [comment]
It is, isn't it? :-)

**Comment by adriaanm** [comment]
Epic!! Congratulations to you all!

One small concern from our end: could you clarify that "production-ready" doesn't imply that Typesafe offers commercial support for scala.js? We may, in the future, but are not planning to offer this as part of our standard agreements just yet.

**Comment by sjrd** [comment]
@adriaanm Right, I forgot that. Done (second paragraph). Feel free to suggest a better wording, though.

**Comment by adriaanm** [comment]
Thanks -- looks good. Would look even better after `s/a product of Typesafe/part of the Typesafe Reactive platform/` (minor) :-)

**Comment by sjrd** [comment]
> Would look even better after `s/a product of Typesafe/part of the Typesafe Reactive platform/` (minor) :-)

Sure. Done.

**Comment by adriaanm** [comment]
Thank you!

**Comment by gkossakowski** [comment]
Congrats! I've read the whole announcement and it sounds great! I'm holding my finger on a tweet button :-)

**Comment by heathermiller** [comment]
Just a heads up – I fixed up the wording here and there to make it sound a bit more natural and made a PR to your scala-lang fork, @sjrd. If you agree/pull it in, the commit should be automatically added to this PR.

Good work by the way! I'm excited by just reading the announcement!

**Pull request merged**